packages/driver/src/cli.mts

```diff
@@ -341,6 +341,7 @@ async function downloadFile(url: string | URL, path: string) {
     throw new Error(`  - Download failed: ${response.statusText}`);
   }
 
+  // @ts-ignore
```

Comment:
Can we use `@ts-expect-error Some note about why` here? We have a lint rule for this, but I wonder if the `.mts` extension is throwing off the matcher. I'll take a look at that.

Reply:
I have no idea why that was there. It seems to be fine without it now.